react/components/EXPERIMENTAL_Table/hooks/useTableMeasures.tsx

```diff
@@ -41,8 +41,10 @@ export function calculateTableHeight(
 }
 
 export function getScrollbarWidth(): number {
-  if (!window || !document || !document.documentElement)
+  const isSSR = typeof document === 'undefined'
```

Comment:
Shouldn't we check if `window` is also not set like that, or it isn't necessary?

```suggestion
  const isSSR = typeof document === 'undefined' || !window
```

Reply:
Done, thanks :smile: